README.md

```diff
@@ -29,7 +29,7 @@ Your answer...
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+router tells controller what action to take depending on the url on the browser.
 ```
```

Comment:
:+1: Yes, although to be explicit I would say depending on the HTTP Request rather than the URL in the browser (since the latter implies that you're dealing with a GET request).